testlib/src/main/java/io/spine/testing/Assertions.java

```diff
@@ -72,6 +90,163 @@ public static UnknownTypeException assertUnknownType(Executable e) {
      */
     @CanIgnoreReturnValue
     public static NullPointerException assertNpe(Executable e) {
+        checkNotNull(e);
         return assertThrows(NullPointerException.class, e);
     }
+
+    /**
+     * Asserts that a condition is true. If it isn't, it throws an
+     * {@link AssertionError} without a message.
+     *
+     * <p>This method is needed to avoid dependency on JUnit 4.x in projects that use
+     * Spine and JUnit5.
+     */
+    @VisibleForTesting
+    static void assertTrue(boolean condition) {
+        if (!condition) {
+            throw new AssertionError();
+        }
+    }
+
+    /**
+     * Asserts that if the passed class has private parameter-less constructor and invokes it
+     * using Reflection.
+     *
+     * <p>Typically this method is used to add a constructor of a utility class into
+     * the covered code.
+     *
+     * <p>Example:
+     * <pre>
+     * public class MyUtilityShould
+     *     ...
+     *     {@literal @}Test
```

Comment:
As long as `{@literal @}` adds an extra space when rendered, could you please move this line one space back?

It should be like this:
```java
      {@literal @}Test
       public void haveUtilityCtor() {
```